kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/WpaSupplicantConfigWriter.java

```diff
@@ -82,18 +85,41 @@ public void visit(NetworkConfiguration config) throws KuraException {
                 .getModifiedNetInterfaceConfigs();
 
         for (NetInterfaceConfig<? extends NetInterfaceAddressConfig> netInterfaceConfig : netInterfaceConfigs) {
-            if (netInterfaceConfig.getType() == NetInterfaceType.WIFI) {
-                // ignore 'mon' interface
-                if (netInterfaceConfig.getName().startsWith("mon.")) {
-                    continue;
-                }
+
+            // skip monitor interface
+            if (netInterfaceConfig.getName().startsWith("mon.")) {
```

Comment:
Same here.

Reply:
i'll keep it, to better explain what we are doing